src/providers/AppLinkProvider.tsx

```diff
@@ -76,28 +80,51 @@ const useAppLink = () => {
         case 'transfer':
           navigator.send({ scanResult: record })
           break
+
+        case 'add_gateway': {
+          if (!qrOnboardEnabled) {
+            if (qrOnboardEnabled === undefined) {
```

Comment:
This check is a little confusing with the one above, can they be combined? When do we expect`qrOnboardEnabled` to be undefined, this is the feature flag right?

Reply:
It will be `undefined` when the qr is scanned from the system camera and the app is closed. We need to wait for the API to respond before deciding to handle the link. This code is just temporary while we're testing it out internally.